iphone/Maps/Bookmarks/Categories/BMCView/BMCViewController.swift

```diff
@@ -47,7 +47,7 @@ final class BMCViewController: MWMViewController {
     // Disable all notifications in BM on appearance of this view.
     // It allows to significantly improve performance in case of bookmarks
     // modification. All notifications will be sent on controller's disappearance.
-    viewModel.setNotificationsEnabled(false)
+    viewModel.setNotificationsEnabled(true)
```

Comment:
Please read the comment. There was a reason to disable notifications. Can it be fixed by enabling only the one necessary notification?

Reply:
I tested it and it works

Reply:
Did you test it with hundreds of bookmark lists with thousands of bookmarks?

Reply:
How does it fix #4740? I am not following...